crates/matrix-synapse/src/lib.rs

```diff
@@ -14,16 +14,28 @@
 
 #![allow(clippy::blocks_in_conditions)]
 
+use anyhow::{bail, Context};
+use error::catch_homeserver_error;
```

Comment:
Can you use `self::` when doing relative imports? I don't like that they get mixed up with libraries

Reply:
seems like this might be fixable by using `rust-analyzer.imports.prefix = "self"` in the rust-analyzer config